### PR TITLE
Updates for 3.16.0

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: source/documentation/io/rdf-input.md [hunk 1]
```diff
@@ -81,24 +80,11 @@ The following is a suggested Apache httpd .htaccess file:
     AddType  application/trix+xml    .trix
     AddType  application/rdf+thrift  .trdf
 
-### Example 1 : Common usage
+### Example 1 : Using the RDFDataMgr {#using-rdfdatamgr}
 
-In this example, a file in the current directory is read as Turtle.
+`RDFDataMgr` provide operations to load, read and write models and datasets.
```
Comment (kinow) [inline source/documentation/io/rdf-input.md:85]: s/provide/provides ?

File: source/documentation/io/rdf-input.md [hunk 2]
```diff
@@ -113,9 +99,20 @@ add data into an existing model or dataset.
     // Read into an existing Model
     RDFDataMgr.read(model, "data2.ttl") ;
 
-### Example 3 : Using RDFParser
+### Example 2 : Common usage {#model-usage}
+
+The original Jena Model API operation for reade and write provide another way to the same machinery:
```
Comment (kinow) [inline source/documentation/io/rdf-input.md:104]: s/reade/read and s/provide/provides ?

File: source/documentation/io/rdf-input.md [hunk 3]
```diff
@@ -137,23 +134,35 @@ The parsers log to a logger called `org.apache.jena.riot`.  To avoid `WARN`
...
+        // Add directory "/tmp" as a place to look for files
+        sm.addLocator(new LocatorFile("/tmp"));
+
+        RDFParser.create()
+            .streamManager(sm)
+            .source("data.ttl")
+            .parse(...);
 
-The `StreamManager` is chosen based on the `Context` object for the
+It can also be set in the chosen based on the `Context` object for the
```
Comment (kinow) [inline source/documentation/io/rdf-input.md:154]: I think it's a bit confusing. Should we define what's the chosen thing/subject is here?

File: source/documentation/notes/stream-manager.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+---
+title: The Jena StreamManager and LocationMapper
+---
+
+The StreamManager is a utility to find and read files into models.
+There is a standard global StreamManager and applications may also
+define specific ones by constructing addition StreamManagers.
```
Comment (kinow) [inline source/documentation/notes/stream-manager.md:7]: s/addition/additional

File: source/documentation/notes/stream-manager.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+- There is no check to see if the alternative system resource is
+  equivalent to the original.
+
+A LocationMapper finds its configuration file by looking for the
+following files, in order:
+
+- `location-mapping.ttl`
+- `location-mapping.rdf`
+- `etc/location-mapping.rdf`
+- `etc/location-mapping.ttl`"
```
Comment (kinow) [inline source/documentation/notes/stream-manager.md:77]: extra " ?

File: source/documentation/notes/stream-manager.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+This is a specified as a path - note the path separator is always
```
Comment (kinow) [inline source/documentation/notes/stream-manager.md:79]: s/This is a specified/This is specified

File: source/documentation/rdfstar/__index.md [hunk 7]
```diff
@@ -70,6 +67,17 @@ is a prefix for `<http://jena.apache.org/ARQ/function#>`.
...
 
+### SPARQL Property Functions related to triple terms
+
+`apf:` is a prefix for `<http://jena.apache.org/ARQ/property#>`.
+
+| Property Function | Description |
+| -------- | ----------- |
+| `<< s p o >> apf:find t` . | Match the triple term. Any `s`, `p`, `o`, and `t` can be RDF terms or variables.|
+
+`apf:find` will result in all the variables being are set accorind to the match,
```
Comment (kinow) [inline source/documentation/rdfstar/__index.md:78]: will result in all the variables being are set accorind to the match

will result in all the variables being set according to the match

?

**Comment by kinow** [comment]
A few comments with possible typos I think, but looks good! Looking forward to 3.16!

**Comment by afs** [comment]
@kinow - thanks for the comments!

**Pull request merged**